### DISCUSSION TITLE
DDR4 SDRAM - Understanding Timing Parameters

### DISCUSSION THREAD

**Discussion by subbdue** [subbdue/systemverilog.io#4] [Website Comments]
Please use this Discussions thread for questions and comments on the article `DDR4 SDRAM - Understanding Timing Parameters`

**Comment by ChoiSeu** [comment]
hello
First of all, I would like to thank you for the great article.

I was just studying through it and had a question that prompted me to write. 

As I understand it, parameters starting with t are time and parameters without t are clock cycles, correct? Or do parameters that start with t also mean clock cycle?

thank u

**Comment by ABMorley** [comment]
I might have missed something but.... In a read cycle the memory device drives the data onto the data bus after a rising edge of DQS... at what point does the ASIC sample it?
